paper-toggle-button.js

```diff
@@ -109,6 +110,11 @@ Polymer({
         @apply --paper-toggle-button-unchecked-button;
       }
 
+      :host(:dir(rtl)) .toggle-button {
```

Comment:
Does this not need a similar workaround to https://github.com/PolymerElements/paper-toggle-button/pull/142/files#diff-1cef05d826161c956d4fc799d670fff1R110?